package.json

```diff
@@ -21,44 +19,37 @@
     "register-service-worker": "^1.7.2",
     "vue": "^2.7.16",
     "vue-class-component": "^7.2.6",
-    "vue-i18n": "^8.28.2",
     "vue-property-decorator": "^9.1.2",
     "vue-router": "^3.6.5",
     "vuetify": "^2.7.2",
     "vuex": "^3.6.2"
   },
   "devDependencies": {
+    "@intlify/unplugin-vue-i18n": "^6.0.0",
     "@intlify/vue-i18n-loader": "^1.1.0",
     "@mdi/js": "^5.9.55",
     "@types/jest": "^24.9.1",
     "@types/lodash-es": "^4.17.12",
     "@typescript-eslint/eslint-plugin": "^4.33.0",
     "@typescript-eslint/parser": "^4.33.0",
-    "@vue/cli-plugin-babel": "~4.5.19",
-    "@vue/cli-plugin-e2e-cypress": "~4.5.19",
-    "@vue/cli-plugin-eslint": "~4.5.19",
-    "@vue/cli-plugin-pwa": "~4.5.19",
-    "@vue/cli-plugin-typescript": "~4.5.19",
-    "@vue/cli-plugin-unit-jest": "~4.5.19",
-    "@vue/cli-service": "~4.5.19",
+    "@vitejs/plugin-vue2": "^2.3.1",
     "@vue/eslint-config-prettier": "^6.0.0",
     "@vue/eslint-config-typescript": "^7.0.0",
     "@vue/test-utils": "^1.3.6",
     "chokidar": "^3.6.0",
     "eslint": "^6.8.0",
     "eslint-plugin-cypress": "^2.15.2",
     "eslint-plugin-prettier": "^3.4.1",
-    "eslint-plugin-vue": "^6.2.2",
+    "eslint-plugin-vue": "^9.31.0",
     "front-matter": "^4.0.2",
     "glob": "^7.2.3",
     "marked": "^3.0.8",
     "prettier": "^2.8.8",
     "sass": "^1.79.4",
-    "sass-loader": "^10.5.2",
     "typescript": "~4.1.5",
-    "vue-cli-plugin-i18n": "~2.1.2",
-    "vue-cli-plugin-vuetify": "^2.5.8",
-    "vue-template-compiler": "^2.7.16",
+    "unplugin-vue-components": "^0.27.4",
+    "vite": "^5.4.11",
+    "vue-i18n": "^8.28.2",
```

Comment:
Should not be a dev dependency

Reply:
Referring to `vue-i18n` here

Reply:
I have migrated this from the dev deps. Thanks for noticing that